common/util_test.go

```diff
@@ -99,3 +123,17 @@ func TestFlushRoutine(t *testing.T) {
 		i++
 	}
 }
+
+func TestGetProjectName(t *testing.T) {
+	testRemote1, err := getMockRepo("https://github.com/ubclaunchpad/inertia.git")
+	assert.Nil(t, err)
+	repoName, err := GetProjectName(testRemote1)
+	assert.Equal(t, "inertia", repoName)
+
+	a, err := getMockRepo("https://github.com/ubclaunchpad/inertia")
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Don't print in tests, since differences aren't automatically caught - use one of the `assert` functions 👍

Reply:
Thanks,

Done.